kiwi/storage/subformat/vagrant_base.py

```diff
@@ -49,20 +49,20 @@ class DiskFormatVagrantBase(DiskFormatBase):
 
     Required methods/variables that child classes must implement:
 
-    - ``provider: str``:
+    * ``provider: str``:
       A static variable or property, should contain the name of the provider.
-      This value is used to create the ``metadata.json`` file and the attribute
-      :attr:`image_format`.
-      (Note: you also must add the image format to
-      :func:`kiwi.defaults.Defaults.get_disk_format_types`)
+      This value is used to create the ``metadata.json`` file and the
+      attribute :attr:`image_format`.
```

Comment:
Ups, this is left over from my first draft, I though that I fixed the documentation. `image_format` must be set explicitly by the child class and is not influenced by `self.provider`.

Reply:
Thanks, will adapt the doc texts and check on the sphinx output

Reply:
done